pkg/scheduler/cache/util.go

```diff
@@ -17,43 +17,20 @@ limitations under the License.
 package cache
 
 import (
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"volcano.sh/volcano/pkg/apis/utils"
+	"volcano.sh/volcano/pkg/apis/helpers"
 	"volcano.sh/volcano/pkg/scheduler/api"
 )
 
-const (
-	shadowPodGroupKey = "volcano/shadow-pod-group"
-)
-
-func shadowPodGroup(pg *api.PodGroup) bool {
-	if pg == nil {
-		return true
-	}
-
-	_, found := pg.Annotations[shadowPodGroupKey]
-
-	return found
-}
-
 func createShadowPodGroup(pod *v1.Pod) *api.PodGroup {
-	jobID := api.JobID(utils.GetController(pod))
-	if len(jobID) == 0 {
-		jobID = api.JobID(pod.UID)
-	}
+	pgName := helpers.GeneratePodgroupName(pod)
```

Comment:
why do we still need this function?

Reply:
just want to keep the same to get pi.Job, like vk job NewJobInfo here.

![image](https://user-images.githubusercontent.com/48981028/62409973-d1ad3700-b611-11e9-8c74-0afd67d120bd.png)

Reply:
we should also update pod's in podgroup controller for that.

Reply:
? ? in podgroup controller pg name is the same as here.

Reply:
At lease we should rename this function, since the `ShadowPodGroup ` concept has been deprecated.